norminette/rules/check_preprocessor_include.py

```diff
@@ -44,6 +44,8 @@ def run(self, context):
                 filetype = tkns[i].value.split(".")[-1][0]
             except:
                 filetype = ""
+        if (i == len(tkns)):
```

Comment:
In Python, we typically do not use parentheses around the if conditions.
```suggestion
        if i == len(tkns):
```

Reply:
Sorry, it became a habit after many C projects following the norm.